### PR TITLE
March 2020 Update

### PR DESCRIPTION
**Pull request by crpietschmann**
This is the PR for the March 2020 Update to the MCW. Issue #40 has the description of what's been done.

### CODE REVIEW

**Comment by codingbandit** [comment]
@DawnmarieDesJardins - Just a note to ensure that Chris moves URLs to the master branch in the script documents prior to merging these changes into master. Currently they are pointing to this branch for anyone that may be testing the content.

File: Hands-on lab/Before the HOL - Business continuity and disaster recovery.md [hunk 1]
```diff
@@ -142,108 +142,104 @@ Duration: 20 minutes
...
+5. Complete the **Resource group** blade using the following inputs and select **Review + Create** and **Create**.
 
-    -   **Resource group name**: BCDRIaaSSecondarySite
-
-    -   **Subscription**: Select your subscription.
-
-    -   **Resource group location**: Central US (Secondary region)
+    - **Resource group name**: `BCDRIaaSSecondarySite`
+    - **Subscription**: Select your subscription.
+    - **Resource group location**: **Central US** (Secondary region)
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Before the HOL - Business continuity and disaster recovery.md]: @crpietschmann @codingbandit 
Hello!  Please review & advise.  Instructions state Resource group location Central US.  Screenshot has Resource details - Region, and shows East US 2.  Different verbiage, different locations.  Thanks, Dawnmarie

Comment (crpietschmann) [reply Hands-on lab/Before the HOL - Business continuity and disaster recovery.md]: For BCDRIaaSSecondarySite resource group, I'm seeing the markdown text and screenshot both showing "Central US" for the region. This is showing correctly. Just to confirm, is this the correct location you're pointing to?

Comment (DawnmarieDesJardins) [reply Hands-on lab/Before the HOL - Business continuity and disaster recovery.md]: Ack, I'm sorry.  I must have scrolled too far on one screen and was comparing the instructions for the primary site with image for the secondary site.  Disregard, I apparently need to walk away and uncross my eyeballs. :)

**Comment by jonsatchwell** [comment]
I just raised an issue on scripts not pointing to master branch as requested earlier in this thread.  They are still pointing to the march update branch

**Pull request merged**